db/db.go

```diff
@@ -2295,16 +2300,20 @@ func GetWithdrawalsCountForQuery(query string) (uint64, error) {
 	// Check whether the query can be used for a validator, slot or epoch search
 	if uiQuery, parseErr := strconv.ParseUint(query, 10, 64); parseErr == nil {
 		searchQuery := `
-				WHERE w.validatorindex = $1
-					OR block_slot = $1
-					OR (block_slot / $3) = $1
-					OR address_text LIKE ($2 || '%')`
```

Comment:
**question:** Why was the address search removed here? Are there not valid addresses that can be parsed as **uint64**?

Reply:
has not been removed, but we only search for exact matches